### DISCUSSION TITLE
AlDente 1.21

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#907] [General]
- Official launch of the Control MagSafe LED feature! Read more about it on our blog: https://apphousekitchen.com/feature-explanation-control-magsafe-led/

- New beta feature “Schedule” which allows you to schedule AlDente Pro features and run them automatically by configuring tasks yourself. Read more about it on our blog: https://apphousekitchen.com/feature-explanation-schedule/
- New, more intuitive setup interface when launching AlDente the first time.
- The status bar and Magsafe LED now show pause and green when 100% are reached, despite the system still trickle charging to full.
- Small UI redesign.
- The “Show on startup” setting is now available in the free version.
- The discharge button is now invisible when “Automatic Discharge” is enabled.
- Fixed a bug that made most of the pro features inaccessible for users that migrated from Intel to Apple Silicon.
- Updated the Helper Application to improve stability.
- Massively overhauled the helper management system.
- The „Start Charging“ shortcut now behaves the same as the „Top Up“ button does. After the device has been unplugged, the original - charge limit is set again.
- Fixed a bug that caused AlDente to prompt some users for their root password after rebooting the device.

With peace & love from Vienna
David & Matthias

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente-Charge-Limiter/releases/tag/1.21'>AlDente 1.21</a>.</em>

**Comment by DamonHD** [comment]
Hurrah!

I have set tasks up to drop the charge limit to 60% on week days just before peak grid demand time, and back to 80% afterwards!

Looking forward to being able to attach `lowpowermode` true/false to those actions to, an abolish my `cron` entries.

Thank you!

Damon
